liqd_product/assets/scss/components/_aside.scss

```diff
@@ -0,0 +1,17 @@
+.aside {
+    background-color: $bg-secondary;
+    color: contrast-color($bg-secondary);
+    padding: $padding;
+}
+
+.aside__block {
+    border-bottom: 1px solid $border-color;
+}
+
+.aside__block__title {
+    font-size: $font-size-sm;
+    text-transform: uppercase;
+    letter-spacing: 0.1em;
+    text-align: center;
+    font-weight: bold;
```

Comment:
Oh, I don't think they should be. We often need to give same styling to different levels of headings depending on the specific hierarchy on a page. This is why I think that we should not have specific heading styling in `base.scss`.